### PR TITLE
Update the output of "create namespace" and "delete namespace"

### PR DESCRIPTION
**Pull request by pchico83**
Signed-off-by: Pablo Chico de Guzman <pchico83@gmail.com>

Update the info messages of `okteto create namespace` and `okteto delete namespace. Also, if the deleted namespace was the current context, update the context namespace to the personal namespace and show a message to inform the user.

Screenshot 2021-11-05 at 10 01 30

### CODE REVIEW

**Review by ifbyol — COMMENTED** [review]
How is the output of the commands updated with those changes? The image doesn't load for me

**Comment by pchico83** [comment]
@ifbyol I added a screenshot, but it is not uploading... let me try again:

Screenshot 2021-11-05 at 10 01 30

**Comment by pchico83** [comment]
@ifbyol there we go, I also updated the PR description with the screenshot

**Review by jLopezbarb — APPROVED** [review]

**Review by ifbyol — APPROVED** [review]

**Comment by ifbyol** [comment]
thanks!!
